packages/client/src/object/convertWireToOsdkObjects.test.ts

```diff
@@ -65,4 +67,23 @@ describe("convertWireToOsdkObjects", () => {
     expect(emptyAttachment).toBeUndefined();
     expect(emptyAttachmentArray).toBeUndefined();
   });
+
+  it("works even with unknown apiNames", () => {
+    const clientCtx = createClientContext(
+      MockOntology,
+      "https://stack.palantir.com",
+      () => "myAccessToken",
+      "userAgent",
+    );
+
+    const object = {
+      __apiName: "unknown",
+      __primaryKey: 0,
+    } as const;
+    const prototypeBefore = Object.getPrototypeOf(object);
+    convertWireToOsdkObjects(clientCtx, [object]);
+    const prototypeAfter = Object.getPrototypeOf(object);
+
+    expect(prototypeBefore).not.toBe(prototypeAfter);
```

Comment:
Got a little confused stepping through here, so what should the prototype of the unknown object be after it runs through the converter?

Reply:
It'll have a newly assigned prototype which has nothing on it (we can't infer anything for `$link`)
Mostly this test is making sure we don't throw an error in the converter.